src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/SwitchOperationsBolt.java

```diff
@@ -335,6 +339,17 @@ private SwitchConnectionsResponse getSwitchConnections(SwitchConnectionsRequest
         return switchOperationsService.getSwitchConnections(request.getSwitchId());
     }
 
+    private List<GetFlowsPerPortForSwitchResponse> getSwitchFlows(GetFlowsPerPortForSwitchRequest request) {
+        log.debug("getSwitchFlows, API call to getSwitchFlows");
```

Comment:
I am not sure if this log could be misleading, the API call would be in [SwitchServiceImpl.java] isn't it?

Reply:
I think I'll remove it and other logs

Reply:
done